packages/Rakefile

```
@@ -42,6 +42,19 @@ class GroongaPackageTask < PackagesGroongaOrgPackageTask
     ]
   end
 
+  def windows_targets_default
+    [
+      "x86-vs2019",
+      "x86-vs2019-with-vcruntime",
+      "x64-vs2019",
+      "x64-vs2019-with-vcruntime",
+      "x86-vs2017",
+      "x86-vs2017-with-vcruntime",
+      "x64-vs2017",
+      "x64-vs2017-with-vcruntime",
```

How about using exact file names for targets? It will simplify `built_package_url`.

```suggestion
      "#{@package}-#{@version}-x64-vs2019.zip",
      "#{@package}-#{@version}-x64-vs2019-with-vcruntime.zip",
      "#{@package}-#{@version}-x86-vs2019.zip",
      "#{@package}-#{@version}-x86-vs2019-with-vcruntime.zip",
      "#{@package}-#{@version}-x64-vs2017.zip",
      "#{@package}-#{@version}-x64-vs2017-with-vcruntime.zip",
      "#{@package}-#{@version}-x86-vs2017.zip",
      "#{@package}-#{@version}-x86-vs2017-with-vcruntime.zip",
      "#{@package}-#{@version}-x64.exe",
      "#{@package}-#{@version}-x64.zip",
      "#{@package}-#{@version}-x86.exe",
      "#{@package}-#{@version}-x86.zip",
```